cmd/cogs/main.go

```diff
@@ -14,14 +14,14 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
-const CogsVersion = "0.5.0"
+const CogsVersion = "0.6.0"
 
 func main() {
 	usage := `
 COGS COnfiguration manaGement S
 
 Usage:
-  cogs gen <ctx> <cog-file> [--out=<type>] [--keys=<key,>] [--not=<key,>] [-n] [-e]
+  cogs gen <ctx> <cog-file> [options]
```

Comment:
I was messing around with this and ran into an error.

```
$ cogs gen kustomize basic.cog.toml 
kustomize: var3: json is an invalid cfgType
```

Is that my own user error?

Reply:
Nope, that's on me! Good catch. I'm going to add the commands as e2e tests